pkg/minikube/registry/drvs/qemu2/qemu2.go

```diff
@@ -125,7 +125,7 @@ func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
 	switch runtime.GOARCH {
 	case "amd64":
 		qemuMachine = "" // default
-		qemuCPU = ""     // default
+		qemuCPU = "max"  // default
```

Comment:
Is there any documentation url we can read what is max and what are other options? And what are downsides of max vs others?

Reply:
I have added the documentation in the commit message. Happy to add it as a comment above this code if you think that is a better place.

Reply:
can u plz link the URL to the documentation I would like to evaluate the options and figure out what exactly is "max"

Reply:
RHEL9 and therefore the dependent distributions and base containers have
a minimum microarchitechture level of x86-64-v2.[1]

The default cpu level of the qemu driver on an Intel Mac is x86-64-v1
even if the host cpu has the x86-64-v2 capability. Using the cpu type of
max allows qemu to use all the host cpu capabilities that is also
supported by the hypervisor framework.[2]

This was fixed for linux host by changing the default within qemu but it
wasn't changed for MacOS.

[1] https://developers.redhat.com/blog/2021/01/05/building-red-hat-enterprise-linux-9-for-the-x86-64-v2-microarchitecture-level
[2] https://gitlab.com/qemu-project/qemu/-/issues/1686#note_1417714887
[3] https://lists.gnu.org/archive/html/qemu-devel/2022-08/msg04066.html

Reply:
thank you for this documentation link, I like the https://lists.gnu.org/archive/html/qemu-devel/2022-08/msg04066.html link you shared, would u plz put that in the comment of right befire 'max' in the code ? and then it would be good to go

Reply:
@medyagh Thanks for the review! I have added the link as a comment.